{{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/{{cookiecutter.django_default_app_name}}/tests/conftest.py

```diff
@@ -1,4 +1,4 @@
-from typing import Generator
+from collections.abc import Generator
```

Comment:
Didn't know it's the right way to do it! :-O

Reply:
neither did I, but it made sense when I seen that https://peps.python.org/pep-0585/#implementation covers more than list/dict/set/...